hsctl

```diff
@@ -102,6 +103,10 @@ preflight_hs() {
             echo "*** creating directory: ${HOST_SSL_DIR} ***"
             mkdir ${HOST_SSL_DIR};
         fi
+        if [[ ! -d ${HS_LOG_FILES} ]]; then
+            echo "*** creating directory: ${HS_LOG_FILES} ***"
+            mkdir ${HS_LOG_FILES};
```

Comment:
I meant as an option to `mkdir` which will spit out an error if it exist, but i neglected to read this properly and see you're just checking for it ahead of time, so you can disregard this.